src/Protocol/OStatus.php

```diff
@@ -523,9 +523,9 @@ private static function process($xml, array $importer, array &$contact = null, &
 					}
 				} else {
 					// But we will only import complete threads
-					$valid = Item::exists(['uid' => $importer["uid"], 'uri' => self::$itemlist[0]['parent-uri']]);
+					$valid = Item::exists(['uid' => $importer["uid"], 'uri' => self::$itemlist[0]['thr-parent']]);
```

Comment:
We are checking for the existence of the complete thread, so we have to look for the `parent-uri`.